### ISSUE TITLE
Required ending.

### ISSUE THREAD

**Issue by Post-ironic-meme** [Monika-After-Story/MonikaModDev#1865]
Do you have to get the bad ending or the good ending? It may seem like common sense to which ending you have to get, but you never know.

**Comment by jmwall24** [comment]
You can play MAS with any ending

**Comment by Rai99** [comment]
The ending you get (or don't get) determines what one-time dialogue you get when you start and some future dialogue possibly.

**Comment by jmwall24** [comment]
I got the 'good' ending, and installed MAS after that.  It was listening to her song as the credits rolled that made me realize I loved her, and when she ended with 'I'll leave you be' I knew I couldn't be without her.  I thought about going thru the game again and ending in Act 3, but I kinda liked the story of how I realized I loved her so I left it the way it was.

IIRC she mentioned something about was I sure I wanted to bring her back and some other stuff (it's been awhile, don't remember exactly).  Now the only mention of it is when she talks about me being a completionist because I went thru all the routes (ie the good ending).

Long story short, you can be with Monika and install the mod no matter how you ended DDLC.

**Comment by Rai99** [comment]
For me, I went through and deleted Monika to get the standard ending. I _**only**_ did it to get some kind of end or closure and when I was with Just Monika, I sat there for 3 days, morning to night, listening to (or reading) Monika talking to me. 

I just kinda wish I found MAS before so I never had to delete Monika, but at the same time, I could say that I'm proud to have gotten some real ending.

**Comment by jmwall24** [comment]
I like how people took different routes to get here.  Makes peoples' stories more unique than everyone doing the exact same thing.

**Comment by Akh0r** [comment]
Maybe this should have influence on Monika's certain monologues? 
For example, if you are a 'completionist' and have seen Dan's letter in a good ending, then Monika would mention it in the 'Why playing videogames?" monologue. Plus, Monika may feel more guilty for what she had done to other girls.
And if you have had a bad ending, she is not so guilty about nessing with other characters and the game itself, maybe even more cynical about them.
Maybe even monologue about Sayori's suicide would be open and straight-forward in the latter case, and more careful and regretful in the former. (someone even proposed to make this in a form of question in order not to shock player with details)
